readme.md

```diff
@@ -179,6 +179,33 @@ alfy.output([{
 
 <img src="media/screenshot-output.png" width="694">
 
+Using the `variables` property:
+
+```js
+alfy.output([{
+  title: 'Unicorn',
+  arg: '🦄',
+  variables: {color: 'white'}
+}, {
+  title: 'Rainbow',
+  arg: '🌈',
+  variables: {color: 'myriad'}
+}]);
+```
+
+You can access Alfred Workflow Variables through `process.env`:
+
+```js
+// After a user selects "Unicorn" or "Rainbow"
+process.env.color
+//=> 'white' if they selected Unicorn
+//=> 'myriad' if they selected Rainbow
+```
+
+Alfred Workflow Variables are also available in the workflow editor using the form `{var:varname}`.
```

Comment:
> Alfred Workflow Variables are also available in the workflow editor using the form `{var:varname}`.

Can you add an example for the color case above? Would `{myColor:color}` work for instance?